translations/translationtool/src/translationtool.php

```diff
@@ -561,18 +628,68 @@ private function rrmdir($path) {
 
 		rmdir($path);
 	}
+
+	public function log(string $message) {
+		if ($this->verbose == 0) {
+			return;
+		}
+		echo " > " . $message . PHP_EOL;
+	}
+}
+
+// arguments handle
+$task = '';
+$usage = false;
+$verbose = 0;
+$returnValue = true;
+
+$index = 0;
+foreach ($argv as $arg) {
+	$index++;
+	if ($index == 1) {
+		$TOOLNAME = $arg;
+		continue;
+	}
+	switch($arg) {
+		case '-h':
+		case '--help':
+			$usage = true;
+			break;
+		case '-v':
+		case '--verbose':
+			$verbose++;
+			break;
+		case 'create-pot-files':
+		case 'convert-po-files':
+		case 'check-files':
+			$task = $arg;
+			break;
+		default:
+			echo "Unknown command parameter : " . $arg . PHP_EOL;
```

Comment:
this is the "appName" parameter from the below usage sample?

Reply:
But is it even used/working? Can't see it anywhere

Reply:
It is used as "name" property. (eg $this->name.".pot")